### ISSUE TITLE
The filetype is empty instead of no-neck-pain

### ISSUE THREAD

**Issue by tadashi-aikawa** [shortcuts/no-neck-pain.nvim#385]
## Description

When executing the set filetype command in the windows generated by no-neck-pain, the filetype used to be 'no-neck-pain', but now it is empty.

Additionally, I confirmed that this change was caused by the commit at https://github.com/shortcuts/no-neck-pain.nvim/commit/ba7475920e0de811768728589f6c1c7df156c803.

## Steps to reproduce

1. Open any file.
2. Move to the window generated by no-neck-pain.
3. Execute the set filetype command.

Then displays

```
filetype=
```

## Expected behavior

```
filetype=no-neck-pain
```

## Environment

- Neovim version: v0.10.1
- no-neck-pain.nvim version: 1.16.0 `4b099238b3518bfc669cf09df90cbad11540176c`

**Comment by shortcuts** [comment]
Hey @tadashi-aikawa thanks for opening the issue and using the plugin

I thought I had a test case for that, great catch!

I'm currently working on the new major but will also ship the fix on the `v1` branch, tagged as `1.4.2` ASAP!

In case you want to avoid those potential bug fixes, you can target de `1.4.1` version safely

**Comment by shortcuts** [comment]
ohhh it's you that owns https://minerva.mamansoft.net/Home? I usually look for "no-neck-pain.nvim" last month searches in google and always see your website :D

**Comment by tadashi-aikawa** [comment]
@shortcuts 
Thank you for your prompt response! I've confirmed that the issue has been fixed. 👍

I'm not sure what changes will be made, but I'm looking forward to the next major version update. :)

Yes, Minerva is a site where I publish what I've created with PKM. No-neck-pain.nvim is an amazing plugin that solved the problems I had with my ultrawide monitor, so I've featured it on several pages. 😊
